pages/sigmaker.tsx

```diff
@@ -59,15 +58,25 @@ const Sigmaker: NextPage = () => {
     })
   }
 
-  const createInputField = (name: string, value: string, label: string) => (
-    <StyledInputField
+  const handleSubmit = () => {
+    setSignatureData(formData)
+    setFormData({
+      fullName: '',
+      title: '',
+      phone: '',
+      email: ''
+    })
+  }
+
+  const createValidatedInputField = (name: string, value: string, label: string) => (
+    <TextValidator
```

Comment:
To increase the input width, you'd want to create something like StyledTextValidator in sigmaker.styles.ts and set the width there. Then use StyledTextValidator here instead. I think a width of 275 or 280 would be pretty similar to what we had before! You can look at the way other components are styled for reference:)